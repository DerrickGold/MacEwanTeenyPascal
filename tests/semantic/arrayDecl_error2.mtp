(* Declare an array with constant containing string*)

const index := 'Stringing';
var something : array(index) of integer;

begin

end.