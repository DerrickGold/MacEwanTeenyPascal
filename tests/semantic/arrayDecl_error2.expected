
Semantic Error [line 4]: 
	Invalid array declaration: 
	Expected 'Constant Integer' index type; found type: 'Constant String'
